controller/volume_controller.go

```diff
@@ -859,18 +859,34 @@ func (c *VolumeController) ReconcileEngineReplicaState(v *longhorn.Volume, es ma
 			continue
 		}
 
-		if status.State == engineapi.ProcessStateComplete && v.Status.CloneStatus.State != longhorn.VolumeCloneStateCompleted {
-			v.Status.CloneStatus.State = longhorn.VolumeCloneStateCompleted
-			c.eventRecorder.Eventf(v, corev1.EventTypeNormal, constant.EventReasonVolumeCloneCompleted,
-				"finished cloning snapshot %v from source volume %v",
+		if !isCloneTargetCopyInProgress(v) {
+			// No longer need to sync up with the engine
```

Comment:
> // No longer need to sync up with the engine

This comment is a bit brief. Could you add some detail to explain why we don’t need to sync with the engine anymore?